TheMovieDb/Modules/MovieDetail/View/DetailHeaderView.swift

```diff
@@ -100,7 +100,7 @@ final class DetailHeaderView: UICollectionReusableView {
         guard let movie = movie else { return }
         let urlImage = MovieConst.imageCDN + (movie.backdropPath ?? (movie.posterPath ?? ""))
         let url = URL(string: urlImage )
```

Comment:
all this logic that creates the path and URL should be moved into another layer, view should only need to work with info provided, also you can return either URL or Data from network request

Reply:
fixed with a viewModel